themes/finna2/templates/Helpers/linkedeventstabs.phtml

```diff
@@ -47,8 +47,8 @@
       </span>
       <span class="event-date-container">
         <span class="event-date-title"><?= $this->transEsc('Time Period'); ?>:</span>
-        <label for="event-date-start" class="sr-only"><?=$this->transEsc('Date');?></label>
-        <input id="event-date-start" class="event-date-start event-datepicker"> &ndash; <label for="event-date-end" class="sr-only"><?=$this->transEsc('Date');?></label><input id="event-date-end" class="event-date-end event-datepicker">
+        <label for="event-date-start" class="sr-only"><?=$this->transEsc('event_date_start');?></label>
```

Comment:
Kun näitä nyt muokataan, niin voisiko korjata id:t käyttämään alaviivoja samalla? Erityisen hämäävää, kun on sama id ja luokka.